public/index.html

```diff
@@ -11,7 +11,7 @@
 </blockquote>
 
 <figure>
-  <iframe frameborder='0' allowfullscreen src='//www.youtube.com/embed/A5ad52AogJ8'></iframe>
+  <video controls poster='https://i.ytimg.com/vi/A5ad52AogJ8/hqdefault.jpg' src='https://redirector.googlevideo.com/videoplayback?pl=38&requiressl=yes&expire=1488314167&initcwndbps=3183750&sparams=dur%2Cei%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&id=o-APGSy7E5MRkK8k3rCVlSMS_hWAgyi5IAa3d0Mgv7TUsR&ei=14q1WL_CCcvn8gSWha6wDQ&ip=2001%3A19f0%3A5%3A1de%3A5400%3Aff%3Afe4f%3A2207&mv=m&mt=1488292474&ms=au&dur=2670.085&ratebypass=yes&mn=sn-ab5l6nly&mm=31&source=youtube&signature=4207F30A68002B5BCF3FC5E13594C46BE1912737.264D5676FA8336C7DCC89B84CB585B4176ACB07B&lmt=1479420772195346&upn=UIhm7t-PIi4&ipbits=0&key=yt6&itag=22&mime=video%2Fmp4'></video>
```

Comment:
i'm pretty sure those are allowed as-is but code highlighting tools are not well updated for html5 because everything changed